### PR TITLE
Add contract details to LeasesCard

### PR DESCRIPTION
**Pull request by ahie**

### CODE REVIEW

File: src/features/customerView/leasesCard/BerthLeasesCard.tsx [hunk 1]
```diff
@@ -22,6 +23,7 @@ const BerthLeasesCard = ({ leases, handleShowContract }: BerthLeasesCardProps) =
       mooringType: lease.mooringType,
       link: lease.harbor ? `/harbors/${lease.harbor.id}` : undefined,
       address: [lease.harbor?.name || '', lease.pierIdentifier || '', lease.berthNum].filter(Boolean).join(' '),
+      renderContractDetails: (leaseId: string) => <BerthContractDetailsContainer leaseId={leaseId} />,
```
Comment (Nizar-Rahme) [inline src/features/customerView/leasesCard/BerthLeasesCard.tsx:26]: If `handleShowContract` is no longer needed, you might consider removing it from `BerthLeasesCardProps`.

**Review by Nizar-Rahme — CHANGES_REQUESTED** [review]
One comment, otherwise, looks good 👍

File: src/features/customerView/leasesCard/BerthLeasesCard.tsx (hunk 1 above)
Comment (ahie) [reply src/features/customerView/leasesCard/BerthLeasesCard.tsx:26]: Done

**Review by Nizar-Rahme — APPROVED** [review]
💎

**Pull request merged**